static/tests/frontend/specs/_utils.js

```diff
@@ -36,6 +36,12 @@ ep_comments_page_test_helper.utils = {
     thisPlugin.commentIcons.timeToUpdateIconPosition = 0;
   },
 
+  setTimeoutOfLineChangeScheduler: function(timeout) {
+    var thisPlugin = helper.padChrome$.window.pad.plugins.ep_comments_page;
+    thisPlugin.commentIcons.timeToUpdateIconPosition = timeout;
+    thisPlugin.commentIcons._addListenersToUpdateIconsPositions();
```

Comment:
acho que essa última linha não precisa...pq teve que chamar de novo?

Reply:
Parece que no momento em que eu defino o timeout maior no teste, o scheduler já foi instanciado com o timeout default. Pra forçar o timeout maior, eu defini o timeout e chamei esse método pra recriar a instância do scheduler com o novo timeout.

Reply:
blz